### DISCUSSION TITLE
I aproved for student pack, but can not use the co-pilot

### DISCUSSION THREAD

**Discussion by Zada5** [community/community#125838] [Copilot Conversations]
### Select Topic Area

General

### Body

I am a student at the university. 
I submitted a request for a GitHub education pack, and I was in the mail and on the GitHub approved! I Can see the approval website. 
I tried to connect to GitHub co-pilot - but it is not working! every time I tried there was the message:  "I do not have a co-pilot account." 
I looked in the QNA, and there was no answer to that. can anyone help? 
Thanks

**Comment by davevad93** [comment]
Hi @Zada5 ,

If you are using the `Copilot` extension in `VS Code` make sure you are logged in with the GitHub account that has student benefits, if you're indeed connected with the right account try logging out and logging back in, if that still doesn't fix it you should contact support by creating a ticket [here](https://support.github.com/contact/account)
